README.md

```diff
@@ -17,30 +17,38 @@ curl -fsSL https://raw.githubusercontent.com/containers/ramalama/s/install.sh |
 You can `list` all models pulled into local storage.
 
 ```
-ramalama list
+$ ramalama list
+NAME                                                                MODIFIED     SIZE
+ollama://tiny-llm:latest                                            16 hours ago 5.5M
+huggingface://tiny-vicuna-1b.q2_k.gguf                              23 hours ago 460M
```

Comment:
Thanks for adding, can we just remove this line:

`huggingface://tiny-vicuna-1b.q2_k.gguf`

?

That was from an older version of ramalama which didn't include the repo in the directory structure. This was a mistake, because people could have identically named .gguf files in two different repos. FWIW the directory structure is like this:

```
lrwxrwxrwx. 1 curtine curtine 96 Jul 25 21:41 /home/curtine/.local/share/ramalama/models/ollama/moondream:latest -> ../../repos/ollama/blobs/sha256:e554c6b9de016673fd2c732e0342967727e9659ca5f853a4947cc96263fa602b
lrwxrwxrwx. 1 curtine curtine 96 Jul 26 12:59 /home/curtine/.local/share/ramalama/models/ollama/granite-code:3b -> ../../repos/ollama/blobs/sha256:02ab8cd2f51476c9a13778448ec83ebbd327ec1f5746c992e3e3453a43ee9aeb
lrwxrwxrwx. 1 curtine curtine 96 Jul 30 12:46 /home/curtine/.local/share/ramalama/models/ollama/granite-code:latest -> ../../repos/ollama/blobs/sha256:02ab8cd2f51476c9a13778448ec83ebbd327ec1f5746c992e3e3453a43ee9aeb
lrwxrwxrwx. 1 curtine curtine 48 Jul 31 11:58 /home/curtine/.local/share/ramalama/models/huggingface/tiny-vicuna-1b.q2_k.gguf -> ../../repos/huggingface/tiny-vicuna-1b.q2_k.gguf
lrwxrwxrwx. 1 curtine curtine 96 Jul 31 18:42 /home/curtine/.local/share/ramalama/models/ollama/tiny-llm:latest -> ../../repos/ollama/blobs/sha256:3682bcf66f4ca1bb217c640490ef0b63265e459bca7dac8799bb671939652421
```

Can catch in a follow on PR also

Reply:
To be honest I'm still don't think I'm handling huggingface right yet, for the case where there is two .gguf files, same name different repo. Need to add a test for that, opening an issue.

Ollama support seems fine though.